docs/frameworks/symfony.md

```diff
@@ -202,35 +157,48 @@ framework:
 If using Webpack Encore, you also need to add the following config at the end of `webpack.config.js`
 
 ```js
+// webpack.config.js
+
 if (Encore.isProduction()) {
     // Note the '/build' at the end of the URL
     Encore.setPublicPath('https://<bucket-name>.s3.amazonaws.com/build');
     Encore.setManifestKeyPrefix('build/')
 }
 ```
 
-Finally, you can compile assets for production in the `public` directory, then synchronize that directory to a S3 bucket:
+Then, you can compile assets for production in the `public` directory, and synchronize that directory to a S3 bucket:
 
 ```bash
 php bin/console assets:install --env prod
 # if using Webpack Encore, additionally run
 yarn encore production
-aws s3 sync public/ s3://<bucket-name>/ --delete --exclude index.php
+aws s3 sync public/ s3://<bucket-name>/ \
+  --delete \
+  --exclude index.php \
+  --exclude public/build/manifest.json \
+  --exclude public/build/entrypoint.json
 ```
 
-### Assets in templates
+Finally, you need to update the `serverless.yml` file to exclude the `assets` and `public/build` directories from deployment:
 
-For the above configuration to work, assets must be referenced in templates via the `asset()` helper:
+```yaml
+# serverless.yml
 
-```html
-<script src="{{ asset('js/app.js') }}"></script>
+package:
+  pattern:
+    - '!assets/**'
+    - '!public/build/**'
+    - 'public/build/manifest.json'
+    - 'public/build/entrypoint.json'
 ```
 
-If your templates reference some assets via direct path, you should edit them to use the `asset()` helper:
+### Assets in templates
+
+For the above configuration to work, assets must be referenced in templates via the `asset()` helper:
 
-```html
+```diff
 - <img src="/images/logo.png">
```

Comment:
Shouldn't it be the following ?

```suggestion
- <img src="/build/images/logo.png">
```

Reply:
After thought, I think we can totally remove this part because Symfony recommends to use the Asset component to display images anyway: https://symfony.com/doc/current/templates.html#linking-to-css-javascript-and-image-assets

Reply:
Well there are two ways to see this :
1. this is common knowledge that you should use `asset()` so no need to document it
2. some users might not know this and this small bit of docs will help them and avoid support from the Bref community for these users 

From my personal experience, there are some users that don't know about this and just use absolute paths in their templates.

So I would keep it.

It's also the same in the laravel docs https://bref.sh/docs/frameworks/laravel.html#assets-in-templates

But maybe linking to that Symfony doc for details about this could be a good idea.

Reply:
It makes sense and remains consistent with the Laravel doc indeed, PR updated 😉